plugins/CustomVariables/VisitorDetails.php

```diff
@@ -41,7 +41,11 @@ public function extendActionDetails(&$action, $nextAction, $visitorDetails)
         for ($i = 1; $i <= $maxCustomVariables; $i++) {
             if (!empty($action['custom_var_k' . $i])) {
                 $cvarKey                 = $action['custom_var_k' . $i];
-                $cvarKey                 = static::getCustomVariablePrettyKey($cvarKey);
+
+                if (in_array($cvarKey, ['_pk_scat', '_pk_scount'])) {
+                    continue; // ignore old site search variables
+                }
```

Comment:
Is it currently supported to send these custom variables manually in tracking requests? Would this break that use case?

Reply:
Actually it seems it wasn't restricted to send those variables manually. Not sure if that is something we should handle 🤔 Don't think someone used those variables accidentally.

Reply:
I guess our options are:

* just stop handling them
* report a tracking error if found so users can fix their tracking code
* keep handling them in some way (ie, not recording as custom var, but as search dimension value)

What do you think?

cc @tsteur

Reply:
I reckon we can keep the logic there as is and doubt indeed someone uses these values. And in worst case these values would just not be visible in the visitor log.